src/services/bundlers/bundlerSwitcher.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable class-methods-use-this */
+
+import { BUNDLER } from '../../consts/bundlers'
+import { Network } from '../../interfaces/network'
+import { Bundler } from './bundler'
+import { getBundlerByName, getDefaultBundler } from './getBundler'
+
+export class BundlerSwitcher {
+  protected network: Network
+
+  protected bundler: Bundler
+
+  protected usedBundlers: BUNDLER[] = []
+
+  // a function to retrieve the current sign account op state
+  protected getSignAccountOpStatus: Function
+
+  // TODO:
+  // no typehints here as importing typehints from signAccountOp causes
+  // a dependancy cicle. Types should be removed from signAccountOp in
+  // a different file before proceeding to fix this
+  protected noStateUpdateStatuses: any[] = []
```

Comment:
Simply move `SigningStatus` and `noStateUpdateStatuses` to `signAccountOp/helper` or create a new `constants` file